### PR TITLE
Maven central

### PR DESCRIPTION
**Pull request by pnowojski**
Added gradle tasks for releasing artifacts to maven central repository

### CODE REVIEW

File: tempto-core/build.gradle [hunk 1]
```diff
@@ -74,3 +108,56 @@ license {
...
+        licenses {
+          license {
+            name 'The Apache Software License, Version 2.0'
+            url 'http://www.apache.org/license/LICENSE-2.0.txt'
+          }
+        }
+        developers {
+          developer {
+            id 'bdch'
+            name 'Boston Development Center for Hadoop'
```
Comment (losipiuk) [inline tempto-core/build.gradle]: Not "Boston Development Center for Hadoop" but "Teradata Center for Hadoop"

File: tempto-core/build.gradle [hunk 2]
```diff
@@ -74,3 +108,56 @@ license {
...
+nexusStaging {
+  packageGroup = group
+  stagingProfileId = '176117bc76f90b'
+}
+
+signing {
+  sign configurations.archives
+}
+
+uploadArchives {
```
Comment (losipiuk) [inline tempto-core/build.gradle]: we need to upload also other artifacts.

File: build.gradle [hunk 3]
```diff
@@ -101,12 +101,17 @@ ext.tempto_logging_log4j = project(':tempto-logging-log4j')
 ext.tempto_examples = project(':tempto-examples')
 ext.expected_result_generator = project(':expected-result-generator')
 ext.third_party_deps = project(':third-party-deps')
-ext.tempto_version = '1.0.28-SNAPSHOT' // changed by arosa
-ext.publish_maven_repository = {
-  url 'http://phabricator.td.teradata.com:8081/nexus/content/repositories/test-framework'
-  credentials {
-    username 'automation'
-    password 'hadoopdb'
+ext.tempto_version = '1.0.29-SNAPSHOT' // changed by pnowojski
```
Comment (kokosing) [inline build.gradle]: I think that once it is open sourced releases have go through maven central. So we need to distinguish between change commits and release commits (where only version is changed). 

Simply, please revert.

File: build.gradle [hunk 4]
```diff
@@ -101,12 +101,17 @@ ext.tempto_logging_log4j = project(':tempto-logging-log4j')
...
+
+
+ext.publish_repositories = {
```
Comment (kokosing) [inline build.gradle]: is it supposed to be removed?

File: tempto-core/build.gradle [hunk 5]
```diff
@@ -40,23 +51,46 @@ dependencies {
...
 }
 
-task sourceJar(type: Jar) {
+// project information
+group 'com.teradata.tempto'
+archivesBaseName = 'teradata-tempto'
+version tempto_version
+
+javadoc {
+  failOnError = false
```
Comment (kokosing) [inline tempto-core/build.gradle]: why it fails, maybe we should fix that and not ignore?

File: tempto-core/build.gradle [hunk 6]
```diff
@@ -74,3 +108,56 @@ license {
...
+        }
+        licenses {
+          license {
+            name 'The Apache Software License, Version 2.0'
+            url 'http://www.apache.org/license/LICENSE-2.0.txt'
+          }
+        }
+        developers {
+          developer {
+            id 'bdch'
```
Comment (kokosing) [inline tempto-core/build.gradle]: tdch

**Comment by kokosing** [comment]
Does it work? Have you upload anything to central?

File: tempto-core/build.gradle (hunk 6 above)
Comment (KBP-TDC) [inline tempto-core/build.gradle]: I would rather avoid TDCH which stands for Teradata Connector for Hadoop (the Ryan's project). TCH? :)

File: build.gradle (hunk 3 above)
Comment (losipiuk) [inline build.gradle]: I do not get your comment . Why revert?

File: tempto-core/build.gradle (hunk 5 above)
Comment (losipiuk) [inline tempto-core/build.gradle]: Log ticket for that

File: tempto-core/build.gradle (hunk 6 above)
Comment (losipiuk) [inline tempto-core/build.gradle]: THC? ;)

Comment (losipiuk) [inline tempto-core/build.gradle]: test reply by email

**Pull request merged**